kubevela.yaml

```diff
@@ -1,13 +1,15 @@
 package:
   name: kubevela
   version: "1.10.3"
-  epoch: 1
+  epoch: 2
   description: KubeVela is a modern application delivery platform that makes deploying and operating applications across today's hybrid, multi-cloud environments easier, faster and more reliable
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
       - ca-certificates-bundle
+      - vela-cli
+      - vela-core
```

Comment:
```suggestion
```

Keeping just `vela-cli` for now will mean anything using this main `kubevela` will be an effective no-op. Once things are updated to use `vela-cli`, then we can remove this entirely